src/components/Accordion/Accordion/Accordion.module.scss

```diff
@@ -0,0 +1,23 @@
+@import "../../../styles/themes";
+@import "../../../styles/typography";
+@import "../../../styles/global-css-settings";
+
+.accordion {
+  height: fit-content;
+  border: 1px solid;
+  @include theme-prop(border-color, ui-border-color);
+  box-sizing: border-box;
+  border-radius: $border-radius-small;
+}
+
+:global(.accordion-item) :global(.expand-collapse--wrapper) :global(.expand-collapse) {
+  border-radius: 0;
```

Comment:
it's a bit weird because expand collapse and accordion should look the same (only the behavior is different a bit)
can you show to meirav how its looks in expand-collapse vs accordion and ask if its ok to apply the changed to expand-collapse

Reply:
Hmm, not sure what u mean...
We don't have any other usages of `ExpandCollapse` other than in `AccordionItem`, but `ExpandCollapse` is a published component, so it can be used by someone outside of Vibe - therefore I think we must be careful changing these overrides and transmitting them to ExpandCollapse e.g.

and without these overrides it'll look like this (the border), so I guess there is no need to consult with Meirav 
![image](https://user-images.githubusercontent.com/104433616/190354496-d2fdcc5e-e24b-4a02-835d-eb58875ae48b.png)